sender.go

```diff
@@ -333,7 +334,7 @@ func (s *LineSender) Int64Column(name string, val int64) *LineSender {
 // Column name cannot contain any of the following characters:
 // '\n', '\r', '?', '.', ',', ”', '"', '\\', '/', ':', ')', '(', '+',
 // '-', '*' '%%', '~', or a non-printable char.
-func (s *LineSender) Long256Column(name, val string) *LineSender {
+func (s *LineSender) Long256Column(name string, val *big.Int) *LineSender {
```

Comment:
This method lacks validation for both non-negative integers and range (256-bit).

Reply:
Yes, and I am not sure how we can perform that before entering the function. If you could suggest any?
we can definitely add it in documentation.

Reply:
The validation has to be in the function body. I left the comment here since it applies to the function, not because it has to be done before the function is called.